packages/@okta/i18n/src/properties/login.properties

```diff
@@ -509,30 +511,38 @@ enroll.webauthn.instructions.noSupportForBiometric = Note: Some browsers may not
 authfactor.webauthn.error = You are currently unable to use a Security key or biometric authenticator. Try again.
 
 ## Enter Passcode Form
-enroll.totp.enterCode = Enter code displayed from the application
+enroll.totp.enterCode = Enter temporary code generated from the application
 
 ## Barcode View
 # {0} is the name of a MFA factor
 enroll.totp.setupApp = Launch {0} on your mobile device and select “Add an account”. Scan the QR code to continue.
 # {0} is the name of a MFA factor
 enroll.totp.setupGoogleAuthApp = Launch {0}, tap the "+" icon, then select "Scan barcode".
 enroll.totp.cannotScan = Can't scan?
-enroll.totp.aria.cannotScan = Setup without scanning a QR code.
+enroll.totp.setupWithoutQRCode = Setup without scanning a QR code.
 enroll.totp.refreshBarcode = Refresh code
 
 ## Manually Setup Push
-enroll.totp.cannotScanBarcode = Can't scan barcode?
+enroll.totp.moreOptions = More options
 enroll.totp.manualSetupInstructions = To set up manually enter your Okta Account username and then input the following in the Secret Key Field
 enroll.totp.manualSetupInstructions.generic = To set up manually enter your Account username and then input the following in the Secret Key Field
 # {0} maps the the configured 'brandName', like "Okta".
 enroll.totp.manualSetupInstructions.specific = To set up manually enter your {0} Account username and then input the following in the Secret Key Field
 enroll.totp.sharedSecretInstructions = Enter your Okta Account username and enter the following in the Secret Key Field
-enroll.totp.sharedSecretInstructions.generic = Enter your Account username and enter the following in the Secret Key Field
-# {0} maps the the configured 'brandName', like "Okta".
-enroll.totp.sharedSecretInstructions.specific = Enter your {0} Account username and enter the following in the Secret Key Field
```

Comment:
Just to confirm - are we OK with this is breaking change? Customers who have customizations for this key will break on a non-major version update.

Reply:
@jmelberg-okta  - we had to make some updates to the OV enrollment flow. can you elaborate on what is non-major version update?

When do we usually have the major version update?

@mauriciocastillosilva-okta  what is your take on this ?

Reply:
Also is there a way to inform them about these changes during non-major version update ?